ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/admin/UserManagementClient.scala

```diff
@@ -0,0 +1,111 @@
+package com.daml.ledger.client.services.admin
+
+import com.daml.ledger.api.domain
+import com.daml.ledger.api.domain.{UserId, User, UserRight}
+import com.daml.ledger.api.v1.admin.user_management_service.UserManagementServiceGrpc.UserManagementServiceStub
+import com.daml.ledger.api.v1.admin.user_management_service.{GetUserRequest, Right}
```

Comment:
Probably renaming `Right`  to `ApiRight` in the import would avoid confusion with the `Right` constructor of `Either`.

Reply:
Or `UserRight`

Reply:
There already is a `UserRight` in scope.

Reply:
Fixed.